operator/cmd/mesh/root.go

```diff
@@ -47,12 +47,14 @@ If set to true, the user is not prompted and a Yes response is assumed in all ca
 This flag can be specified multiple times to overlay multiple files. Multiple files are overlaid in left to right order.`
 	installationCompleteStr  = `Installation complete`
 	ForceFlagHelpStr         = `Proceed even with validation errors.`
+	verificationHelpStr      = `Verify after installation`
```

Comment:
Verify the Istio control plane after installation.

Reply:
As mentioned in another comment, this is not needed due to duplication.